src/backend/distributed/master/master_metadata_utility.c

```diff
@@ -57,11 +61,274 @@ static void RecordDistributedRelationDependencies(Oid distributedRelationId,
 												  Node *distributionKey);
 static ShardPlacement * TupleToShardPlacement(TupleDesc tupleDesc,
 											  HeapTuple heapTuple);
+static List * ShardIntervalsOnWorkerNode(WorkerNode *workerNode, Oid relationId);
+static StringInfo GenerateSizeQueryOnMultiplePlacements(Oid distributedRelationId,
+														List *shardIntervalList,
+														char *sizeQuery);
+static uint64 DistributedTableSizeOnWorker(WorkerNode *workerNode, Oid relationId,
+										   char *sizeQuery);
+static uint64 DistributedTableSize(Oid relationId, char *sizeQuery);
+static void ErrorIfNotSuitableToGetSize(Oid relationId);
 
 
 /* exports for SQL callable functions */
 PG_FUNCTION_INFO_V1(master_stage_shard_row);
 PG_FUNCTION_INFO_V1(master_stage_shard_placement_row);
+PG_FUNCTION_INFO_V1(citus_table_size);
```

Comment:
Function bodies of citus size functions came before master_stage_shard_row and master_stage_shard_placement_row. So it would be better if we put their declarations first.